hw/spinal/config.scala

```diff
@@ -0,0 +1,15 @@
+import spinal.core
+
+package object config {
+  import core._
+
+  val spinal = SpinalConfig(
+    targetDirectory = "hw/gen",
+    defaultConfigForClockDomains = ClockDomainConfig(
+      resetActiveLevel = LOW
```

Comment:
Do we realy want to ahve resetActiveLow as a default ?

Reply:
In my studies and in the company I were in, all resets were active low (maybe it is because of ASIC stuff).

Also, an added option can be removed easier than it can be added XD

Reply:
Instead of removing the line, we could set it to HIGH explicitly in this file? So that users don't have to find it in the docs?

It is the default configuration file, we can populate with often used keys :smiley:

Reply:
The issue i have with it is that people will just use that config without realy looking at it, and be surprised it isn't  following the SpinalHDL default.

> Instead of removing the line, we could set it to HIGH explicitly in this file? So that users don't have to find it in the docs?

Yes, that's a good alternative.

Reply:
Wilco

Reply:
Done